labs/RivePlayer/src/RivePlayer.Platform.cs

```diff
@@ -0,0 +1,106 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#if WINDOWS_UWP || (WINUI2 && HAS_UNO_WASM)
```

Comment:
```suggestion
#if WINDOWS_UWP || (WINUI2 && HAS_UNO)
```

Think this should just be `HAS_UNO`?

Reply:
You should be able to shorten this to just:
```suggestion
#if !WINAPPSDK
```

Reply:
`!WINAPPSDK` seemed to build locally for WinUI2 and WinUI3 both. Let's see if it works here!